tox.ini

```diff
@@ -22,3 +25,8 @@ commands =
     generate_validation: python setup.py generate_validation
     # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
     ansible_syntax: ansible-playbook --syntax-check playbooks/byo/config.yml
+
+    # Unset GOPATH because tests use relative imports. This should be removed if
+    # we require openshift-ansible to live in a Go work space and use absolute
+    # imports in tests (desirable).
+    integration: env -u GOPATH go test -v ./test/integration/...
```

Comment:
Hmm, I failed to see how that would be used?

Did you mean something like

```
[testenv:integration]
setenv=
    GOPATH=
```
?

But then, that looks like a lot more to unset an envvar than using `env -u`?

I'm unsetting `GOPATH` just to not require the code to be within GOPATH, and use relative imports. Relative imports are notably a bad idea in Go, I just wanted to experiment and see how it would compare. This basically affects CI in that we need to decide if the checkout path is within a Go work space or not. And we need to decide between Travis and Jenkins...